### DISCUSSION TITLE
Feature request: par2 checksum

### DISCUSSION THREAD

**Discussion by igreenman** [doublecmd/doublecmd#1089] [Suggestions]
### Description

It would be nice to add the creation and use of par2 checksums.

**Comment by alexx2000** [comment]
Do you mean [Parchive](https://parchive.github.io/)? It is not a simple checksum, it is a redundant file format. I guess it is possible to attach **par2cmdline** as [external arhiver](https://doublecmd.github.io/doc/en/multiarc.html#archivers) to Double Commander.
